lib/generators/solidus_subscriptions/install/templates/app/controllers/concerns/create_subscription.rb

```diff
@@ -0,0 +1,15 @@
+module CreateSubscription
```

Comment:
Nit: The frozen string literal magic comment is missing.

Reply:
Rubocop is complaining as well! 🤖